internal/nodes/dequeue_ingestion.go

```diff
@@ -154,6 +154,11 @@ func DequeueIngestion(key string, fullsync bool) {
 		svcNames, svcFound := schema.GetParentServices(name, namespace, key)
 		if svcFound && utils.CheckIfNamespaceAccepted(namespace) {
 			for _, svcNSNameKey := range svcNames {
+				svcNSName := strings.Split(svcNSNameKey, "/")
+				if svcObj, _ := utils.GetInformers().ServiceInformer.Lister().Services(svcNSName[0]).Get(svcNSName[1]); !lib.ValidateSvcforClass(key, svcObj) {
+					utils.AviLog.Warnf("key: %s, msg: not processing Service %s for %s %s", key, svcNSNameKey, objType, name)
```

Comment:
Can we mention reason of rejection (not valid LB Class)? It will be easier for debugging

Reply:
I've added LBClass invalidation reason in logs